tests/dfns.rs

```diff
@@ -0,0 +1,95 @@
+#[cfg(test)]
+mod e2e {
+    use api::runtime_types::tangle_primitives::services::field::Field;
+    use api::services::calls::types::call::Args;
+    use blueprint_test_utils::test_ext::*;
+    use blueprint_test_utils::*;
+    use cargo_tangle::deploy::Opts;
+    use dfns_cggmp21_blueprint::keygen::KEYGEN_JOB_ID;
+    use gadget_sdk::error;
+    use gadget_sdk::info;
+    use gadget_sdk::tangle_subxt::tangle_testnet_runtime::api;
+
+    pub fn setup_testing_log() {
+        use tracing_subscriber::util::SubscriberInitExt;
+        let env_filter = tracing_subscriber::EnvFilter::from_default_env();
+        let _ = tracing_subscriber::fmt::SubscriberBuilder::default()
+            .without_time()
+            .with_target(true)
+            .with_span_events(tracing_subscriber::fmt::format::FmtSpan::NONE)
+            .with_env_filter(env_filter)
+            .with_test_writer()
+            .finish()
+            .try_init();
+    }
+
+    #[tokio::test(flavor = "multi_thread")]
+    #[allow(clippy::needless_return)]
+    async fn keygen() {
+        setup_testing_log();
```

Comment:
Can we just make all of this default apart of the new_test_ext? It's seems totally unnecessary to make the user define this every test. They can just override it if they want, no?

Reply:
For `setup_testing_log()`, we can move that to `blueprint-test-utils`, but the user needs to opt into only one subscriber. It gets set globally and can't be overridden.

As for the other setup, it can probably be made default.